tools/ci/cibuild.sh

```diff
@@ -204,7 +204,23 @@ function elf-toolchain {
     esac
   fi
 
-  x86_64-elf-gcc --version
+  command x86_64-elf-gcc --version
+}
+
+function util-linux {
```

Comment:
a bit strange name used. I understand that it comes from ubuntu component, but anyway strange to read that Darwin target needs to install `util-linux`.
Nit: Maybe we can rename this to `flock` or `flock-util`? Just brainstorming here

Reply:
Just googled https://www.howtodojo.com/flock-command-not-found/ and it says that `brew install util-linux` should be working on MAC. Can we try that instead of getting a new tap?

Reply:
Ok, but let's merge this patch first to unblock ci, I will provide patch later.

Reply:
> a bit strange name used. I understand that it comes from ubuntu component, but anyway strange to read that Darwin target needs to install `util-linux`. Nit: Maybe we can rename this to `flock` or `flock-util`? Just brainstorming here

i think `flock` function name will cause the following `if ! type flock` to related to bash function itself, resulting in an incorrect conditional result.

Reply:
I'm fine with `util-linux`, but let's try if `brew install util-linux` works instead of tap

Reply:
@pkarashchenko here is the patch: https://github.com/apache/nuttx/pull/10629